apps/twitch-bot/internal/command/invite.go

```diff
@@ -6,29 +6,27 @@ import (
 	"strings"
 
 	"github.com/gempir/go-twitch-irc/v3"
-	"github.com/senchabot-dev/monorepo/apps/twitch-bot/client"
 	"github.com/senchabot-dev/monorepo/apps/twitch-bot/internal/models"
-	"github.com/senchabot-dev/monorepo/apps/twitch-bot/server"
 )
 
-func InviteCommand(client *client.Clients, server *server.SenchabotAPIServer, message twitch.PrivateMessage, commandName string, params []string) {
+func (s *commands) InviteCommand(message twitch.PrivateMessage, commandName string, params []string) {
```

Comment:
[for follow up pr]

let's change this to use the first letter of the related struct:

```suggestion
func (c *commands) InviteCommand(message twitch.PrivateMessage, commandName string, params []string) {
```